src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.h

```diff
@@ -8,12 +8,10 @@ struct BgKin2Bombwall;
 typedef void (*BgKin2BombwallActionFunc)(struct BgKin2Bombwall*, GlobalContext*);
 
 typedef struct BgKin2Bombwall {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x64];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderCylinder collider;  
     /* 0x01A8 */ BgKin2BombwallActionFunc actionFunc;
-    /* 0x01AC */ char unk_1AC[0x4];
+    /* 0x01AC */ s8 unk_1AC[0x4];
```

Comment:
unk_1AC  looks like it is a single s8 that is used as a timer. The rest of this is most likely struct padding which you don't need to be explicit about.
```suggestion
    /* 0x01AC */ s8 timer;
```